spec/supplejack/extraction/enrichment_execution_spec.rb

```diff
@@ -79,10 +79,6 @@
 
       let(:subject) { described_class.new(sample_job) }
 
-      it 'does not create TransformationJobs for failed pages' do
-        expect { subject.call }.not_to change(TransformationJob, :count)
-      end
```

Comment:
Should we have an equivalent test? Maybe I missed it

Reply:
This is covered by the checking of if there are TransformationWorkers enqueued in Sidekiq below :)